_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,344 @@ rules.
+istio-ingress-service-account to access ingress key/cert.       
+
+1. Update RBAC set up for istio-pilot-service-account and istio-mixer-istio-service-account.
+
+   Delete ClusterRoleBinding istio-pilot-admin-role-binding-istio-system and 
+   istio-mixer-admin-role-binding-istio-system. 
+   Also delete ClusterRole istio-mixer-istio-system.  
+
+   ```bash
+   kubectl delete ClusterRoleBinding istio-pilot-admin-role-binding-istio-system
+   kubectl delete ClusterRoleBinding istio-mixer-admin-role-binding-istio-system
+   kubectl delete ClusterRole istio-mixer-istio-system
+   ```
+   List all secrets in namespace istio-system that we need to protect using RBAC.
+   
+   ```bash
+   kubectl get secrets -n istio-system
+   ```
+   This produces the following output:
+   
+   ```bash
+   NAME                                        TYPE                                  DATA      AGE
+   default-token-4wwkb                         kubernetes.io/service-account-token   3         7d
+   istio-ca-secret                             istio.io/ca-root                      2         7d
+   istio-ca-service-account-token-rl4xm        kubernetes.io/service-account-token   3         7d
+   istio-egress-service-account-token-vbfwf    kubernetes.io/service-account-token   3         7d
+   istio-ingress-certs                         kubernetes.io/tls                     2         7d
+   istio-ingress-service-account-token-kwr85   kubernetes.io/service-account-token   3         7d
+   istio-mixer-service-account-token-29qbb     kubernetes.io/service-account-token   3         7d
+   istio-pilot-service-account-token-t6kmf     kubernetes.io/service-account-token   3         7d
+   istio.default                               istio.io/key-and-cert                 3         7d
+   istio.istio-ca-service-account              istio.io/key-and-cert                 3         7d
+   istio.istio-egress-service-account          istio.io/key-and-cert                 3         7d
+   istio.istio-ingress-service-account         istio.io/key-and-cert                 3         7d
+   istio.istio-mixer-service-account           istio.io/key-and-cert                 3         7d
+   istio.istio-pilot-service-account           istio.io/key-and-cert                 3         7d
+   ```
+   Create a file istio-pilot-mixer-istio-system.yaml, which defines a new ClusterRole that grants  
+   access permission to kubernetes.io/service-account-token, istio.io/key-and-cert, and  
+   istio.io/ca-root types of resource instances. This file also defines a ClusterRoleBinding to  
+   bind this new ClusterRole to service accounts istio-pilot-service-account and 
+   istio-mixer-istio-service-account.
+   
+   ```bash
+   kind: ClusterRole
+   apiVersion: rbac.authorization.k8s.io/v1beta1
+   metadata:
+     name: istio-pilot-mixer-istio-system
+   rules:
+   - apiGroups: [""] # "" indicates the core API group
```

Comment:
Same here and below. Please avoid showing unrelated content.

Reply:
Done.